NerdyPy/NerdyPy.py

```diff
@@ -12,10 +12,12 @@
 import configparser
 from pathlib import Path
 from datetime import datetime
+
```

Comment:
Warum ist da ein Leerschritt?

Reply:
War dir Zeilenumbruch zu Mainstream? ^^

Ich hab kA, habe die imports nicht manuell ausgeführt.

Ist das jetzt ne Aufforderung das zu Ändern?

Reply:
Wenn man auf die Datei in PyCharm "Rechtsklick -> Reformate Code -> Optimize Imports" macht wird das alles automatisch so formatiert und ist dementsprechen das standard verhalten für neue Imports